payments-core/src/main/java/com/stripe/android/paymentsheet/injection/PaymentSheetCommonModule.kt

```diff
@@ -0,0 +1,66 @@
+package com.stripe.android.paymentsheet.injection
+
+import androidx.core.os.LocaleListCompat
+import com.stripe.android.BuildConfig
+import com.stripe.android.Logger
+import com.stripe.android.networking.StripeApiRepository
+import com.stripe.android.networking.StripeRepository
+import com.stripe.android.payments.core.injection.ENABLE_LOGGING
+import com.stripe.android.payments.core.injection.IOContext
+import com.stripe.android.paymentsheet.analytics.DefaultDeviceIdRepository
+import com.stripe.android.paymentsheet.analytics.DefaultEventReporter
+import com.stripe.android.paymentsheet.analytics.DeviceIdRepository
+import com.stripe.android.paymentsheet.analytics.EventReporter
+import com.stripe.android.paymentsheet.repositories.CustomerApiRepository
+import com.stripe.android.paymentsheet.repositories.CustomerRepository
+import com.stripe.android.paymentsheet.repositories.StripeIntentRepository
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+import kotlinx.coroutines.Dispatchers
+import javax.inject.Named
+import javax.inject.Singleton
+import kotlin.coroutines.CoroutineContext
+
+@Module
+internal abstract class PaymentSheetCommonModule {
```

Comment:
This module could be combined with `PaymentCommonModule`, there will be 3 entry point of the SDK, `FlowController`, `PaymentSheet` and `PaymentLauncher`, the first two will use `PaymentCommonModule`.

Reply:
`PaymentCommonModule` will stay in `payments-core`, `PaymentSheetCommonModule` is for `paymentsheet` module classes.